assets/css/frontend.scss

```diff
@@ -403,6 +403,10 @@ a.sensei-certificate-link {
 	}
 }
 
+.sensei-item-no-display {
```

Comment:
Thanks for the suggestion @donnapep ! I've implemented the changes here https://github.com/Automattic/sensei/pull/7675/commits/f9207011b9273647ad18fc69c9147685c840a54e .

Initially I used a more generic class and placed it in a catch-all file because I was envisioning it a bit like `bootstrap` or `tailwind`'s [general purpose class](https://tailwindcss.com/docs/display) to hide an element. To avoid having `display: none` for many different classes in many files. But I've changed it now.

An additional note is that we didn't have any specific stylesheet that loads only for single Quizzes. So I've added one, and adding this style only there. Styles for quizzes used to go into some catch-all files. I haven't moved those quiz specific styles in this PR to avoid risk of breaking and also that'd be out of scope for this PR. But moving the quiz specific styles from general stylesheet to this new quiz specific stylesheet will probably be a good idea once we are done with bugs and important enhancements. Should I create an issue for that?

Reply:
> Should I create an issue for that?

Sure!